pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -19,6 +19,13 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/kubeflow/tf-operator/pkg/apis/mxnet/validation"
+
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
 	"reflect"
```

Comment:
Can we move this to Line 21?

Reply:
em.. not sure why my autosave plugin doesn't work. I will manually fix imports

Reply:
`goimports -w pkg/controller.v1/mxnet/mxjob_controller.go` interesting, output is same as first revision. 

After I change "reflect" to line 21 and run the command, it still doesn't update the file.. 

Seem the tool thinks both are valid results. However, we definitely move to line 21 is elegant.  Looks like an improvement item on `goimports`